pandas/core/frame.py

```diff
@@ -6983,7 +6983,8 @@ def value_counts(
         dropna: bool = True,
     ) -> Series:
         """
-        Return a Series containing counts of unique rows in the DataFrame.
+        Return a Series containing the number of times each unique row occurs
```

Comment:
Our code check notes that this should fit on one line

Reply:
This issue should be fixed with the most recent commit and now also adhere to the 88 character limit for lines.